lib/verification/verify.go

```diff
@@ -93,6 +107,10 @@ func (v *VerificationModule) VerifyOnPremAWSManagedDeployment(configFile string)
 	if err != nil {
 		return err
 	}
+	configVerify := &config_verify.ConfigVerifyImpl{}
```

Comment:
line num 110 to 113 & 124 to 127 & 158 to 161 are duplicates. You can write a common method

Reply:
made a different function for that.